mock_hvca_test.go

```diff
@@ -594,9 +667,11 @@ func mockTrustChain(w http.ResponseWriter, r *http.Request) {
 
 // mockNotImplemented is a stub handler that writes a 501 not implemented
 // response.
+/* This is commented out because the linter is complaining >:(
 func mockNotImplemented(w http.ResponseWriter, r *http.Request) {
```

Comment:
is this still used ? if not we cannot remove it because the linter is complaining?

Reply:
Alternatively: what's the complain the linter is making?  Maybe it can be resolved if this code is useful

Reply:
This was used to mock unimplemented functions. I'd rather  it just stick around since  its one of those puzzle pieces thats nice to have lying around. Otherwise we remove it, and then when we need to mock another endpoint we rewrite it.

Reply:
To answer the question, the linter is complaining about an unused function.

Reply:
I think better to remove it, ideally we would not mock any endpoint in the future like this